src/diff/index.js

```diff
@@ -165,7 +165,11 @@ export function diff(
 			tmp = c.render(c.props, c.state, c.context);
 			let isTopLevelFragment =
 				tmp != null && tmp.type == Fragment && tmp.key == null;
-			newVNode._children = isTopLevelFragment ? tmp.props.children : tmp;
+			newVNode._children = isTopLevelFragment
+				? tmp.props.children
+				: Array.isArray(tmp)
+				? tmp
+				: [tmp];
```

Comment:
There are scenario's where we skip an update with a placeholder, in this case tmp returns null with render, this is not an array so we `[null]` it